lib/stemcell/templates/bootstrap.sh.erb

```diff
@@ -145,16 +142,21 @@ configure_chef() {
   echo "$environment" > $envfile
   chmod 644 $envfile
 
+  echo "$username" > $launchedbyfile
+  chmod 644 $launchedbyfile
+
   echo "configuring chef solo..."
   cat<<EOF > ${chef_dir}/solo.rb
-repo_dir =    "${repo_dir}"
-cookbook_path "#{repo_dir}/cookbooks"
-role_path     "#{repo_dir}/roles"
+repo_dir = "${repo_dir}"
+
+cookbook_path ["#{repo_dir}/site-cookbooks", "#{repo_dir}/cookbooks"]
+role_path "#{repo_dir}/roles"
 data_bag_path "#{repo_dir}/data_bags"
 ssl_verify_mode :verify_peer
+log_level :info
+log_location STDOUT
 
-log_level        :info
-log_location     STDOUT
+Ohai::Config[:plugin_path] << "#{repo_dir}/ohai_plugins"
```

Comment:
I was wondering what would happen if the `chef/ohai_plugins` directory does not exist. I checked Ohai source code for [version 7.4](https://github.com/chef/ohai/blob/7.4-stable/lib/ohai/loader.rb#L41) and the [latest](https://github.com/chef/ohai/blob/be251f7b9d1d835fc3b0158f2cd8595d17451811/lib/ohai/loader.rb#L42). In both cases it will be fine. The latest version of Ohai will log a warning though but I think it's actually useful information and optionally adding the directory to Ohai config will only create for room for confusion.